### PR TITLE
section commit for ERMS-1878

### PR DESCRIPTION
**Pull request by croqueGrec09**
further refactor of subscription - license relation

### CODE REVIEW

File: app/grails-app/views/survey/currentSurveysConsortia.gsp [hunk 1]
```diff
@@ -367,15 +369,15 @@
...
+                                    params="[surveyConfigID: surveyConfig?.id]"
                                     class="ui icon button">
                                 <i class="icon blue chart pie"></i>
                                 </div>
                             </g:link>
                         </g:else>
                     </td>
                     <td>
-                        <g:link controller="survey" action="show" id="${surveyInfo.id}" params="[surveyConfigID: surveyConfig.id]" class="ui button icon">
+                        <g:link controller="survey" action="show" id="${surveyInfo.id}" params="[surveyConfigID: surveyConfig?.id]" class="ui button icon">
```
Comment (klodav) [inline app/grails-app/views/survey/currentSurveysConsortia.gsp:380]: valid url without surveyConfigID ?

File: app/grails-app/views/survey/currentSurveysConsortia.gsp [hunk 2]
```diff
@@ -367,15 +369,15 @@
                         </g:if>
                         <g:else>
                             <g:link controller="survey" action="surveyEvaluation" id="${surveyInfo.id}"
-                                    params="[surveyConfigID: surveyConfig.id]"
+                                    params="[surveyConfigID: surveyConfig?.id]"
```
Comment (klodav) [inline app/grails-app/views/survey/currentSurveysConsortia.gsp:372]: valid url without surveyConfigID ?

Comment (croqueGrec09) [reply app/grails-app/views/survey/currentSurveysConsortia.gsp]: reverted in b4064a9

File: app/grails-app/views/survey/currentSurveysConsortia.gsp (hunk 1 above)
Comment (croqueGrec09) [reply app/grails-app/views/survey/currentSurveysConsortia.gsp]: reverted in b4064a9

**Pull request merged**